rackcity/views/asset_views.py

```diff
@@ -1442,217 +959,178 @@ def asset_bulk_export(request):
     csv_writer = csv.DictWriter(csv_string, fields)
     csv_writer.writeheader()
     csv_writer.writerows(serializer.data)
-    return JsonResponse(
-        {"export_csv": csv_string.getvalue()},
-        status=HTTPStatus.OK,
-    )
+    return JsonResponse({"export_csv": csv_string.getvalue()}, status=HTTPStatus.OK,)
 
 
-@api_view(['POST'])
+@api_view(["POST"])
 @permission_classes([IsAuthenticated])
 def network_bulk_upload(request):
     data = JSONParser().parse(request)
-    if 'import_csv' not in data:
+    if "import_csv" not in data:
         return JsonResponse(
             {
-                "failure_message":
-                    Status.IMPORT_ERROR.value +
-                    BulkFailure.IMPORT.value,
-                "errors":
-                    "Bulk upload request should have a parameter 'import_csv'"
+                "failure_message": Status.IMPORT_ERROR.value + BulkFailure.IMPORT.value,
+                "errors": "Bulk upload request should have a parameter 'import_csv'",
             },
-            status=HTTPStatus.BAD_REQUEST
+            status=HTTPStatus.BAD_REQUEST,
         )
-    base_64_csv = data['import_csv']
-    csv_bytes_io = BytesIO(
-        b64decode(re.sub(".*base64,", '', base_64_csv))
-    )
-    csv_string_io = StringIO(csv_bytes_io.read().decode('UTF-8-SIG'))
-    csvReader = csv.DictReader(csv_string_io)
+    base_64_csv = data["import_csv"]
+    csv_bytes_io = BytesIO(b64decode(re.sub(".*base64,", "", base_64_csv)))
+    csv_string_io = StringIO(csv_bytes_io.read().decode("UTF-8-SIG"))
+    csv_reader = csv.DictReader(csv_string_io)
     expected_fields = BulkNetworkPortSerializer.Meta.fields
-    given_fields = csvReader.fieldnames
-    if (
-        len(expected_fields) != len(given_fields)  # check for repeated fields
-        or set(expected_fields) != set(given_fields)
-    ):
+    given_fields = csv_reader.fieldnames
+    if len(expected_fields) != len(given_fields) or set(  # check for repeated fields
+        expected_fields
+    ) != set(given_fields):
         return JsonResponse(
             {
-                "failure_message":
-                    Status.IMPORT_ERROR.value +
-                    BulkFailure.IMPORT_COLUMNS.value
+                "failure_message": Status.IMPORT_ERROR.value
+                + BulkFailure.IMPORT_COLUMNS.value
             },
-            status=HTTPStatus.BAD_REQUEST
+            status=HTTPStatus.BAD_REQUEST,
         )
     bulk_network_port_datas = []
-    for row in csvReader:
+    for row in csv_reader:
```

Comment:
6. Renamed some variables, changing CP to _cp and decomissioned to decommissioned